src/main/java/mate/controller/driver/DriverDeleteController.java

```diff
@@ -0,0 +1,25 @@
+package mate.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.DriverService;
+
+@WebServlet(value = "/drivers/delete")
```

Comment:
you should choose - or use web.xml confi for all controllers or WebServlet annotation, don't mix them